apps/web/pages/availability/[schedule].tsx

```diff
@@ -100,6 +105,7 @@ export default function Availability() {
             }
           : undefined
       }
+      travelSchedules={isDefaultSchedule ? travelSchedules || [] : []}
```

Comment:
@calcom/platform I assume I also need to add the travelSchedules to PlatformAvailabilitySettingsWrapper now?